source/documentation/other-topics/aws-rds-migration.md

```diff
@@ -128,26 +115,8 @@ psql -U target_username \
 
 If using a local file is problematic, those two commands can be piped together (`|`)  
 
-
-
-#### Step 3 [ DMS ONLY ]  
-
-This step has to be done with the assistance of the Cloud Platform.
-
-The DMS stack is build using a terraform module. 
-
-Please refer to [this](https://github.com/ministryofjustice/cloud-platform-terraform-dms)
- repository to see the DMS module instructions:  
- [https://github.com/ministryofjustice/cloud-platform-terraform-dms](https://github.com/ministryofjustice/cloud-platform-terraform-dms)
-
-
-IF YOU HAVE FOLLOWED THAT STEP, GO STRAIGHT TO STEP 4
-
 #### Step 3 [ PURE PG_DUMP ]
```

Comment:
You don't need `[ PURE PG_DUMP ]` here anymore, since you're only talking about one migration method.

Reply:
fixed. thank you.